BepInEx.Core/Utility.cs

```diff
@@ -221,6 +221,54 @@ public static bool TryResolveDllAssembly<T>(AssemblyName assemblyName,
         return false;
     }
 
+    /// <summary>
+    ///     Try to resolve the symbols data from an assembly DLL.
+    /// </summary>
+    /// <param name="dllPath">File path of the DLL assembly.</param>
+    /// <param name="symbolsData">The symbols of the DLL assembly.</param>
+    /// <returns>True, if the symbols were found. Otherwise, false.</returns>
+    public static bool TryResolveAssemblySymbols(string dllPath, out byte[] symbolsData)
+    {
+        symbolsData = null;
```

Comment:
nit (don't consider this blocking): assigning the out parameter in `TryGet*`  methods right before the `return` instead at the start will force you to think about the return value in all possible code paths

Reply:
good idea, I ended up doing that